.ci/integrationTestSelector.groovy

```diff
@@ -95,7 +95,9 @@ pipeline {
       }
       environment {
         TMPDIR = "${WORKSPACE}"
-        REUSE_CONTAINERS = "true"
+        ENABLE_ES_DUMP = "true"
+        PATH = "${WORKSPACE}/${BASE_DIR}/.ci/scripts:${env.PATH}"
+        NAME = 'all'
```

Comment:
Is $NAME descriptive enough? Wdyt about $MODULE_NAME?

I know it's a backport and the forward version is in master, so if considered I'd apply it to master too

Reply:
Uhm, it's not really a module though, but something else, `name` sounds generic enough for the current context